source/upgrade.txt

```diff
@@ -104,6 +104,16 @@ and upgrade versions.
 
 .. diagram for this example?
 
+.. _version-4.11-breaking-changes:
+
+Version 4.11 Breaking Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- {+mdb-server+} v3.6 is no longer supported. The minimum supported {+mdb-server+}
+  version is now v4.0.
+
+- The minimum wire version is now 7. See :manual:`minWireVersion </manual/reference/command/hello/#mongodb-data-hello.minWireVersion>`.
```

Comment:
This URL ended up as a 404 for me, I think `/manual` can be omitted from the path.

Reply:
thanks for catching!